python/servo/testing_commands.py

```diff
@@ -90,9 +90,12 @@ def run_test(self, prefix, args=[], release=False):
                           HELP_RENDER_MODE)
     @CommandArgument('--release', default=False, action="store_true",
                      help="Run with a release build of servo")
-    def test(self, params, render_mode=DEFAULT_RENDER_MODE, release=False):
+    @CommandArgument('--faster', default=False, action="store_true",
+                     help="Only check changed files and skip the WPT lint")
+    def test(self, params, render_mode=DEFAULT_RENDER_MODE, release=False, faster=False):
         suites = OrderedDict([
-            ("tidy", {}),
+            ("tidy", {"kwargs": {"faster": faster},
+                     "include_arg": "include"}),
```

Comment:
The spacing of the first character on the second line. The `"` should match up with the `"` above it